types/chart.js/index.d.ts

```diff
@@ -231,6 +233,8 @@ declare namespace Chart {
         fontFamily?: string;
         padding?: number;
         generateLabels?(chart: any): any;
+        filter?(item: ChartLegendItem, data: ChartData): any;
```

Comment:
I’m not sure if the return type should be `any`, but the documentation doesn’t say anything… I’m guessing it casts the return value to boolean, so this is fair enough.